src/app/product/components/product-detail-page/product-details/product-details.component.ts

```diff
@@ -76,40 +77,38 @@ export class ProductDetailsComponent implements OnInit {
     this.calculateInnerWidth();
     this.addMetaInfo(this.product);
     this.initData();
-    this.store.dispatch(this.productsActions.getRelatedProduct(this.productID));
-    this.relatedProducts$ = this.store.select(relatedProducts);
-    this.store.dispatch(this.productsActions.getProductReviews(this.productID));
-    this.reviewProducts$ = this.store.select(productReviews);
-    this.findBrand();
-    this.addJsonLD(this.product);
+    // this.store.dispatch(this.productsActions.getRelatedProduct(this.productID));
```

Comment:
Remove code if not required, no need for having commented code. Also, in case this is required put a TODO tag.